src/ugenecl/src/Main.cpp

```diff
@@ -163,6 +168,20 @@ static void setDataSearchPaths() {
     } else if (QDir(appDirPath + RELATIVE_DEV_DATA_DIR).exists()) {    //data location for developers
         printf("ADDED PATH %s \n", qPrintable(appDirPath + RELATIVE_DEV_DATA_DIR));
         dataSearchPaths.push_back(appDirPath + RELATIVE_DEV_DATA_DIR);
+#ifdef Q_OS_MAC
+    } else {
+        CFURLRef appUrlRef = CFBundleCopyBundleURL(CFBundleGetMainBundle());
```

Comment:
Is it the same as BundleInfoMac::getDataSearchPath() ?

Reply:
done